src/components/shared/Validations.tsx

```diff
@@ -18,49 +21,54 @@ export const Validations: React.FunctionComponent<IValidations> = ({ required, v
     </div>
   );
 
-  return validationCount ? (
-    <Popover
-      boundary="window"
-      interactionKind="hover"
-      content={
-        <div className="p-5" style={{ maxHeight: 500, maxWidth: 400 }}>
-          {Object.keys(validations).map((key, index) => {
-            const validation = validations[key];
+  return (
+    <>
+      {deprecated ? <span className="ml-2 text-darken-7 dark:text-lighten-6">deprecated</span> : null}
+      {validationCount ? (
+        <Popover
+          boundary="window"
+          interactionKind="hover"
+          content={
+            <div className="p-5" style={{ maxHeight: 500, maxWidth: 400 }}>
+              {Object.keys(validations).map((key, index) => {
+                const validation = validations[key];
 
-            let elem = null;
-            if (Array.isArray(validation)) {
-              elem = validation.map((v, i) => (
-                <div key={i} className="mt-1 mr-1 flex items-center">
-                  <div className="px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded">{String(v)}</div>
-                  {i < validation.length - 1 ? <div>,</div> : null}
-                </div>
-              ));
-            } else if (typeof validation === 'object') {
-              elem = (
-                <div className="m-1 px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded" key={index}>
-                  {'{...}'}
-                </div>
-              );
-            } else {
-              elem = (
-                <div className="m-1 px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded" key={index}>
-                  {JSON.stringify(validation)}
-                </div>
-              );
-            }
+                let elem = null;
+                if (Array.isArray(validation)) {
+                  elem = validation.map((v, i) => (
+                    <div key={i} className="mt-1 mr-1 flex items-center">
+                      <div className="px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded">{String(v)}</div>
+                      {i < validation.length - 1 ? <div>,</div> : null}
+                    </div>
+                  ));
+                } else if (typeof validation === 'object') {
+                  elem = (
+                    <div className="m-1 px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded" key={index}>
+                      {'{...}'}
```

Comment:
Why don't we stringify the validation here too so we can display the properties? Similar to what were doing on L53

Reply:
I think the reason we did it in the first place is that the object might be large and therefore the text produced wouldn't be readable.